### PR TITLE
[FIX] mail: add missing call icon in the discuss sidebar

### PR DESCRIPTION
**Pull request by ThanhDodeurOdoo**
Following the discuss refactoring* the call icon in the discuss sidebar has not been reimplemented.

This commit fixes this issue.

\* https://github.com/odoo/odoo/pull/110188

### CODE REVIEW

**Review by zel-odoo — APPROVED** [review]

**Comment by ThanhDodeurOdoo** [comment]
reimplementation of: https://github.com/odoo/odoo/blob/16.0/addons/mail/static/src/components/discuss_sidebar_category_item/discuss_sidebar_category_item.xml#L37-L39

File: addons/mail/static/src/web/discuss/sidebar.xml [hunk 1]
```diff
@@ -104,6 +104,9 @@
                 t-on-click.stop="() => this.leaveChannel(thread)" title="Leave this channel" role="img"/>
             <div t-if="threadService.canUnpin(thread)" t-attf-class="fa fa-times ms-1" t-on-click.stop="() => threadService.unpin(thread)" title="Unpin Conversation" role="img"/>
         </div>
+        <t t-if="Object.keys(thread.rtcSessions).length > 0">
```
Comment (phenix-factory) [inline addons/mail/static/src/web/discuss/sidebar.xml:107]: This t-if can go on the div itself now

Comment (ThanhDodeurOdoo) [reply addons/mail/static/src/web/discuss/sidebar.xml]: @phenix-factory updated, I think that it has been possible for a while but that it was a guideline in discuss to separate them

**Comment by phenix-factory** [comment]
@ThanhDodeurOdoo Follow up PR could add a test for this, it's doesn't cost much and document the feature.

**Pull request closed without merging**